project_resource_calendar/i18n/fr.po

```diff
@@ -423,72 +438,30 @@ msgid "Service"
 msgstr "Services"
 
 #. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:63
-#: model:ir.ui.view,arch_db:project_resource_calendar.report_events
-#, python-format
-msgid "Sunday"
-msgstr ""
-
-#. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_calendar_event_event_task_id
-msgid "Task"
-msgstr ""
-
-#. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:106
-#, python-format
-msgid ""
-"The resource cannot be double-booked with any overlapping meetings or events."
+#: model:ir.model.fields,field_description:project_resource_calendar.field_weekly_report_wizard_date_start
+msgid "Start Date"
 msgstr ""
 
 #. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:99
-#, python-format
-msgid ""
-"The room cannot be double-booked with any overlapping meetings or events."
-msgstr ""
+#: model:ir.model.fields,field_description:project_resource_calendar.field_weekly_report_wizard_state
+msgid "Status"
+msgstr "Statut"
 
 #. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:60
-#: model:ir.ui.view,arch_db:project_resource_calendar.report_events
-#, python-format
-msgid "Thursday"
-msgstr ""
+#: model:ir.model.fields,field_description:project_resource_calendar.field_calendar_event_event_task_id
+msgid "Task"
+msgstr "Tâche"
 
 #. module: project_resource_calendar
 #: model:ir.ui.view,arch_db:project_resource_calendar.calendar_event_report_view
 msgid "To:"
 msgstr "À:"
 
-#. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:58
-#: model:ir.ui.view,arch_db:project_resource_calendar.report_events
-#, python-format
-msgid "Tuesday"
-msgstr ""
-
 #. module: project_resource_calendar
 #: model:ir.model.fields,field_description:project_resource_calendar.field_resource_calendar_instrument_category_type
 msgid "Type"
 msgstr "Type"
```

Comment:
And why are you deleting these msgid "Tuesday"	
msgstr "" weekdays?

Reply:
@eilst  Changed it